README.md

```diff
@@ -84,32 +84,50 @@ See [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_gui
 
 ## Using this collection
 
-You can call modules by their Fully Qualified Collection Name (FQCN), such as `community.network.routeros_command`.
-The following example task replaces configuration changes in the existing configuration on a network device, using the FQCN:
+You can call modules by their Fully Qualified Collection Name (FQCN), such as `community.network.a10_server`.
+The following example task creates a new a10 server, using the FQCN:
```

Comment:
From the module's description, this might be better:
```suggestion
The following example task creates a new server load balancer object on an A10 Networks device, using the FQCN:
```
(I don't know the module at all though :) )